include/eve/arch/arm/sve/tags.hpp

```diff
@@ -29,7 +29,7 @@ namespace eve
     static constexpr std::size_t expected_cardinal = bytes / sizeof(Type);
 
     template<typename Type>
-    static constexpr std::size_t fundamental_cardinal = 8 / sizeof(Type);
+    static constexpr std::size_t fundamental_cardinal = expected_cardinal<Type>;
```

Comment:
Isn't expected cardinal using this value member ? Sounds a bit circular.

Reply:
No, it doesn't